<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowInput</class>
 <widget class="windowInput" name="windowInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>windowInput</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="centralwidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="LabelExtroversion">
           <property name="minimumSize">
            <size>
             <width>330</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Extroversion</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="LabelAgreableness">
           <property name="minimumSize">
            <size>
             <width>330</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Agreableness</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="LabelConscientiousness">
           <property name="minimumSize">
            <size>
             <width>330</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Conscientiousness</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="LabelNeuroticism">
           <property name="minimumSize">
            <size>
             <width>330</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Neuroticism</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="LabelOpenness">
           <property name="minimumSize">
            <size>
             <width>330</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Openness to Exp.</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="LabelDate">
           <property name="minimumSize">
            <size>
             <width>330</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Date added</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="LabelCountry">
           <property name="minimumSize">
            <size>
             <width>330</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Country</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxExtroversion">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxAgreableness">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBoxConsc">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBoxNeuroticism">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBoxOpenness">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="maxVisibleItems">
            <number>5</number>
           </property>
           <property name="maxCount">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDateEdit" name="dateEdit">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton1">
           <property name="minimumSize">
            <size>
             <width>191</width>
             <height>71</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Go back to the
start screen</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAdd">
           <property name="minimumSize">
            <size>
             <width>191</width>
             <height>71</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>windowInput</class>
   <extends>QWidget</extends>
   <header>windowinput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
